.travis.yml

```diff
@@ -4,13 +4,5 @@ android:
   components:
     - tools
     - platform-tools
-
-    # The SDK version used to compile your project
```

Comment:
I kind of like having these comments to remind me what setting they correspond to.

Reply:
Sure :) I'll revert these line changes